### PR TITLE
Fixes #204: Added easier way to choose emoji

### LINKED ISSUE #204: Emoji Heatmapper
Please add an easier way to chose emoji similar to many other app e.g. Facebook, Google Chat/Gmail etc.
screenshot from 2017-06-21 09-41-25

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #204 
Added emoji dropdown in which it is easy to choose emoji .

**Screenshots for the change:**
image

Demo link: https://kavithaenair.github.io/emojiHeatmapper/

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @Achint08 @SKrPl @daminisatya @mariobehling Please review.

**Comment by singhpratyush** [comment]
@kavithaenair: I am unable to load the page properly and this error comes up - 
screenshot from 2017-06-22 23 10 46

Also, one of the errors is about `ol` being not defined. Please take a look.

**Comment by kavithaenair** [comment]
@singhpratyush you have to **Load unsafe scripts** to see the map

File: emojiHeatmapper/lib/js/util.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
+'use strict';
+
```
Comment (Achint08) [inline emojiHeatmapper/lib/js/util.js]: Just a question. What is this file related to?

Comment (kavithaenair) [inline emojiHeatmapper/lib/js/util.js]: I have used already existing libraries. You can check it [here](https://github.com/OneSignal/emoji-picker). @Achint08
